wsgi.py

```diff
@@ -44,5 +46,12 @@ def about_page():
     return render_template("about.html")
 
 
+@app.route("/activate/<confirmation_token>/")
```

Comment:
Sprawdzałeś co się stanie jak wpiszesz losowy token np: `iyeurvbeuirbgiuerbiurb`?

Reply:
Naprawiłem!
Jak jest gjdijgdijgidfjgidfj to wtedy wyrzuca 404 not found